### PR TITLE
Bug/um 362 relational funnels query

### PR DESCRIPTION
**Pull request by franciscao633**
Refactor all queries to same behavior. Fix MySql params bug.

### CODE REVIEW

File: lib/prisma.ts [hunk 1]
```diff
@@ -19,8 +19,20 @@ const POSTGRESQL_DATE_FORMATS = {
   year: 'YYYY-01-01',
 };
 
-function getAddMinutesQuery(field: string, minutes: number) {
-  const db = getDatabaseType();
+function toUuid(): string {
```
Comment (mikecao) [inline lib/prisma.ts:22]: This function is removed. The rawQuery handles this internally. You can just do {{var::uuid}}.

File: lib/prisma.ts [hunk 2]
```diff
@@ -80,80 +92,72 @@ function getFilterQuery(filters = {}, params = []): string {
...
-  const initParamLength = 3;
-
   return urls.reduce(
     (pv, cv, i) => {
       const levelNumber = i + 1;
-      const start = i > 0 ? ',' : '';
+      const startSum = i > 0 ? 'union ' : '';
 
       if (levelNumber >= 2) {
         pv.levelQuery += `\n
```
Comment (mikecao) [inline lib/prisma.ts:126]: The \n here is not needed.

File: lib/prisma.ts [hunk 3]
```diff
@@ -168,7 +172,8 @@ export default {
   getDateQuery,
   getTimestampIntervalQuery,
   getFilterQuery,
-  getFunnelQuery,
+  toUuid,
   parseFilters,
+  getFunnelQuery,
```
Comment (mikecao) [inline lib/prisma.ts:177]: This should probably be moved inside `getFunnel.ts` since it probably won't be used anywhere else.

**Review by mikecao — APPROVED** [review]
Some minor comments, but approved otherwise.

File: lib/prisma.ts (hunk 1 above)
Comment (franciscao633) [reply lib/prisma.ts]: removed uuid function

File: lib/prisma.ts (hunk 3 above)
Comment (franciscao633) [reply lib/prisma.ts]: move relational / CH function into getFunnel.ts

**Pull request merged**